pygments/lexers/graphviz.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers.graphviz
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the DOT language (graphviz).
+
+    :copyright: Copyright 2006-2020 by the Pygments team, see AUTHORS.
```

Comment:
2021 by now, sorry it took us so long.

Reply:
Well, I didn't expect it being merged in 2020. We have time for that!